apps/etterlatte-behandling/src/main/kotlin/config/BeanFactory.kt

```diff
@@ -269,4 +272,17 @@ class EnvBasedBeanFactory(private val env: Map<String, String>) : CommonFactory(
             minutterGamleHendelser = env.getValue("HENDELSE_MINUTTER_GAMLE_HENDELSER").toLong()
         ).schedule()
     }
+
+    override fun featureToggleService(): FeatureToggleService {
+        return FeatureToggleService.initialiser(
```

Comment:
For the record: Eg ser litt på lure måtar å eventuelt forenkle dette på, kjem har nokre idear om å kombinere application.conf og env, men det kjem eventuelt i ein seinare PR.